### ISSUE TITLE
"Could not obtain access token" error when running example from documentation

### ISSUE THREAD

**Issue by GNazar** [apioo/fusio-sdk-java#1]
Steps:
1. Create simple test app:
```
public class Main {

    public static void main(String[] args) throws ClientException {
        List<String> scopes = new ArrayList<>();
        scopes.add("backend");
        TokenStoreInterface tokenStore = new MemoryTokenStore();
        OAuth2 credentials = new OAuth2("test", "FRsNh1zKCXlB", "https://demo.fusio-project.org/authorization/token", "", tokenStore, scopes);
        Client client = new Client("https://demo.fusio-project.org/", credentials);
        BackendOperationCollection all = client.backend().operation().getAll(0, 16, "");
    }
```
2. Add javasdk version 2.0.0
3. Run and observe an error: 
```
INFO: Could not obtain access token
app.sdkgen.client.Exception.Authenticator.InvalidAccessTokenException: Could not obtain access token
	at app.sdkgen.client.Authenticator.OAuth2Authenticator.parseTokenResponse(OAuth2Authenticator.java:160)
	at app.sdkgen.client.Authenticator.OAuth2Authenticator.request(OAuth2Authenticator.java:184)
	at app.sdkgen.client.Authenticator.OAuth2Authenticator.fetchAccessTokenByRefresh(OAuth2Authenticator.java:124)
	at app.sdkgen.client.Authenticator.OAuth2Authenticator.getAccessToken(OAuth2Authenticator.java:147)
	at app.sdkgen.client.Authenticator.OAuth2Authenticator.getAccessToken(OAuth2Authenticator.java:154)
	at app.sdkgen.client.Authenticator.OAuth2Authenticator.process(OAuth2Authenticator.java:44)
	at org.apache.http.protocol.ImmutableHttpProcessor.process(ImmutableHttpProcessor.java:133)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
	at org.fusioproject.sdk.BackendOperationTag.getAll(BackendOperationTag.java:195)
	at org.fusioproject.sample.Main.main(Main.java:23)

Exception in thread "main" org.fusioproject.sdk.CommonMessageException: The server returned an error
	at org.fusioproject.sdk.BackendOperationTag.getAll(BackendOperationTag.java:204)
```

**Comment by chriskapp** [comment]
Hi @GNazar I have just released a new version 3.0 which contains the latest generated version of the SDK which should fix the problem, you can also take a look at our sample s.
https://github.com/apioo/fusio-sample-java-cli

**Comment by GNazar** [comment]
It was fast.
Thank you!
